pkg/tiller/kind_sorter.go

```diff
@@ -123,5 +123,9 @@ func (k *kindSorter) Less(i, j int) bool {
 	if !ok {
 		return true
 	}
+	if first == second {
+		// same kind so sub sort alphanumeric
+		return a.name < b.name
+	}
```

Comment:
My pleasure. I apologize for the too-late drive by comments.
I haven't been able to stay on top of the PR queue lately, so I only noticed this one by way of mention in the "helm-dev" Slack channel.